### PR TITLE
Remove old migrations code

### PR DESCRIPTION
**Pull request by sqs**
Removes old migrations that no longer need to run. See commit messages.

> This PR does not need to update the CHANGELOG because these are so old (4+ months)

### CODE REVIEW

File: cmd/frontend/db/stores.go [hunk 1]
```diff
@@ -7,7 +7,6 @@ var (
 	DiscussionMailReplyTokens = &discussionMailReplyTokens{}
 	Repos                     = &repos{}
 	Phabricator               = &phabricator{}
-	UserActivity              = &userActivity{} // DEPRECATED: use package useractivity instead (based on persisted redis cache)
```
Comment (sqs) [inline cmd/frontend/db/stores.go]: @dadlerj Confirming OK to remove the DB user activity store and the migration for it to Redis?

**Comment by slimsag** [comment]
We should also actually remove the user_activity table from the prod postgres DB.

File: cmd/frontend/db/stores.go (hunk 1 above)
Comment (slimsag) [reply cmd/frontend/db/stores.go]: https://github.com/sourcegraph/sourcegraph/issues/414#issuecomment-430113880

Comment (dadlerj) [reply cmd/frontend/db/stores.go]: Yep! I had accidentally submitted a deletion of `MigrateUserActivityData` [in the other repo](https://github.com/sourcegraph/enterprise/pull/13711/files#diff-5e1575a1cf42ada21f940efc9405206e) just before this — but it should have been here.

**Comment by sqs** [comment]
Filed the DB removal in https://github.com/sourcegraph/sourcegraph/issues/527

**Pull request merged**